src/c++/perf_analyzer/genai-perf/genai_perf/main.py

```diff
@@ -51,7 +51,7 @@
 
 
 def generate_inputs(args):
-    # TODO: remove once fale support is in
+    # TODO: remove once file support is implemented
```

Comment:
i liked fale. its like fail, but somehow more accurate.
signed, the guy who made the typo